src/_nebari/initialize.py

```diff
@@ -87,39 +88,54 @@ def render_config(
             }
 
     if cloud_provider == ProviderEnum.do:
+        config["digital_ocean"] = {
+            "kubernetes_version": kubernetes_version,
+            "region": region,
+        }
+
         config["theme"]["jupyterhub"][
             "hub_subtitle"
         ] = f"{WELCOME_HEADER_TEXT} on Digital Ocean"
-        if kubernetes_version is not None:
-            config["digital_ocean"] = {"kubernetes_version": kubernetes_version}
```

Comment:
nevermind, my mistake

Reply:
When `kubernetes_version` is passed in as `None`, this tells us that we should default to the latest version we support. 

For this particular module, I wanted to ensure the kubernetes version and region are properly set before returning a config, especially since they are needed for the subsequent deployment.